app/config/config_dev.yml

```diff
@@ -7,6 +7,13 @@ web_profiler:
     toolbar: true
     intercept_redirects: false
 
+monolog:
+    handlers:
+        main:
+            type:  stream
+            path:  %site.path_www%/app/logs/%kernel.environment%.log
+            level: debug
+
```

Comment:
this is not needed in `_dev` because it's in the main config file https://github.com/forkcms/forkcms/blob/master/app/config/config.yml#L27-L32